scripts/travis.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+npm run install:packages
+
+export CLIENT_ID=0oad5a19u0IVnKU1g0h7
+export REDIRECT_URI=http://localhost:8080/implicit/callback
+
+cd packages/jwt-verifier
+npm test > ./jwt-verifier-test.log
+
+if (cat ./jwt-verifier-test.log | grep -i -e "failed:" -e "failing" > /dev/null); then
+    echo "[ERROR] jwt-verifier tests failed!"
+    cat ./jwt-verifier-test.log
+    exit -1
+else
+    echo "jwt-verifier tests successful."
+fi
+
+export CLIENT_ID=0oad5983foWn2Gefr0h7
+export REDIRECT_URI=http://localhost:8080/authorization-code/callback
+
+cd ../oidc-middleware
+npm test > ./oidc-middleware-test.log
+
+if (cat ./oidc-middleware-test.log | grep -i -e "failed:" -e "failing" > /dev/null); then
+    echo "[ERROR] oidc-middleware tests failed!"
+    cat ./oidc-middleware-test.log
+    exit -1
+else
+    echo "oidc-middleware tests successful."
+fi
+
+export CLIENT_ID=0oadwuob65038lE6f0h7
+export REDIRECT_URI=http://localhost:3000/implicit/callback
+cd ../okta-angular
+npm test  > ./okta-angular-test.log
+
+if (cat ./okta-angular-test.log | grep -i "Failed:" > /dev/null); then
+    echo "[ERROR] okta-angular tests failed!"
+    cat ./okta-angular-test.log
+    exit -1
+else
+    echo "okta-angular tests successful."
+fi
+
+cd ../okta-react
+npm test > ./okta-react-test.log
+
+if (cat ./okta-react-test.log | grep -e "FAILED:" -e "Failed:" > /dev/null); then
+    echo "[ERROR] okta-react tests failed!"
+    cat ./okta-react-test.log
+    exit -1
+else
+    echo "okta-react tests successful."
+fi
+
+# Disregard this for review
+# export CLIENT_ID=0oadwuob65038lE6f0h7
+# export REDIRECT_URI=http://localhost:3000/implicit/callback
+
+# cd packages/okta-vue/test/e2e/harness
+# npm install
+# npm install chromedriver@2.33.0 --chromedriver-cdnurl=http://chromedriver.storage.googleapis.com
+# curl https://selenium-release.storage.googleapis.com/3.5/selenium-server-standalone-3.5.0.jar > selenium-server-standalone.jar
+# npm test
```

Comment:
Why is this failing in travis but passing locally? Can you attach logs to this for us to look at please?

Reply:
I read that it's because of some issue with running nightwatch tests on docker container. Will attach the logs.

Reply:
@jmelberg-okta: This is the error I get on travis - 

```
Starting selenium server... started - PID:  6133
[Test] Test Suite
=====================
Running:  redirects to Okta for login when trying to access a protected page
Error retrieving a new session from the selenium server
Connection refused! Is selenium server started?
{ Error: socket hang up
    at createHangUpError (_http_client.js:345:15)
    at Socket.socketCloseListener (_http_client.js:377:23)
    at emitOne (events.js:120:20)
    at Socket.emit (events.js:210:7)
    at TCP._handle.close [as _onclose] (net.js:545:12) code: 'ECONNRESET' }
```

Reply:
Gotcha. Lets investigate this as a follow up.